aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -668,6 +680,8 @@ template <class T> Vectorized<T> inline operator+(const Vectorized<T> &a, const
   return c;
 }
 
+  VECTORIZED_SUPPORT_SCALARS_FOR_BINARY_OP(+)
```

Comment:
What's with the indentation?

Reply:
it's what my editor wanted to do. this file isn't formatted by lintrunner :\

Reply:
sent PR to autoformat everything